src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleArray = sourceString.split(';');
```

Comment:
The variable name 'styleArray' is not very descriptive. Consider a more meaningful name that reflects the contents of the array, such as 'styleDeclarations' or 'cssDeclarations'.